### PR TITLE
Move tox build to Jenkins.

### PR DESCRIPTION
**Pull request by jeschkies**
## High-level description
This moves the tox build onto Jenkins to be executed in parallel.

## Corresponding DC/OS tickets (required)

- [D2IQ-66808](https://jira.d2iq.com/browse/D2IQ-66808)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -26,21 +26,41 @@ pipeline {
...
-	    }
-        }  
+    stage('Build') {
+      parallel {
+        stage('Tox') {
+	  agent {
+	    label 'python-dind'
+	  }
+	  steps {
+	    sh('rm -rf dcos-release.config.yaml')
```
Comment (janisz) [inline Jenkinsfile:36]: How about moving it into Makefile and run in docker container so it will be easier to run tox locally on commit hook?

Comment (jeschkies) [reply Jenkinsfile:36]: Thanks for resurrecting this. Yes, that's a good idea. But we need the credentials first. I created a ticket https://jira.d2iq.com/browse/D2IQ-66808

Comment (jeschkies) [reply Jenkinsfile]: I'll do it as part of #6818

**Comment by jeschkies** [comment]
@mesosphere-mergebot label ready for review

**Comment by jeschkies** [comment]
@mesosphere-mergebot request-review @orsenthil

**Comment by jeschkies** [comment]
@mesosphere-mergebot changelog-not-required internal change

**Comment by jeschkies** [comment]
@mesosphere-mergebot request-review @jongiddy

**Comment by jeschkies** [comment]
@mesosphere-mergebot request-review @pierrebeitz @cgawron1

**Comment by jeschkies** [comment]
@mesosphere-mergebot changelog-not-required internal change

**Pull request merged**